packages/ec-core/src/individual/ec.rs

```diff
@@ -128,20 +297,40 @@ impl<GG> WithScorer for GG {
     }
 }
 
-// G is Genome
-// S is Scorer
-impl<G, D, S> Distribution<EcIndividual<G, S::Score>> for IndividualGenerator<D, S>
+impl<GenomeT, GenomeGeneratorT, ScorerT> Distribution<EcIndividual<GenomeT, ScorerT::Score>>
```

Comment:
I appreciate the expansion of the generic names here, but do we need the `T` at the end of the names? It doesn't really add anything, and it's not a pattern we (I) have used anywhere else.

Reply:
The `T` is really to differentiate the generics from the Traits and ensure no name collisions - my feeling was since you felt it was necessary to add comments for what the generics are just name them as such :)